stock_request/__manifest__.py

```diff
@@ -8,6 +8,7 @@
     "license": "LGPL-3",
     "website": "https://github.com/stock-logistics-warehouse",
     "author": "Eficent, "
+              "Avoin.Systems, "
```

Comment:
Please add yourself to the contributors section, not to authorship. Authorship is granted when your contribution to the module is significant. Good pratices in OCA.

Reply:
Whoops, sorry. Yeah, I'll fix that.

Reply:
OK @jbeficent, fixed.